src/basho_bench_worker_sup.erl

```diff
@@ -60,8 +60,28 @@ init([]) ->
     %% specs for each
 
     basho_bench_profiler:maybe_start_profiler(basho_bench_config:get(enable_profiler, false)),
-    Workers = worker_specs(basho_bench_config:get(concurrent), []),
-    {ok, {{one_for_one, 5, 10}, Workers}}.
+
+    Workers = basho_bench_config:get(workers, []),
+    WorkerTypes = basho_bench_config:get(worker_types, []),
```

Comment:
We should use the one arity config get here so that it explodes in the even workers is set and worker types is not set.